src/components/Settings/Notifications/NotificationsTelegram.tsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import { Field, Form, Formik } from 'formik';
+import useSWR from 'swr';
+import LoadingSpinner from '../../Common/LoadingSpinner';
+import Button from '../../Common/Button';
+import { defineMessages, useIntl } from 'react-intl';
+import axios from 'axios';
+import * as Yup from 'yup';
+import { useToasts } from 'react-toast-notifications';
+
+const messages = defineMessages({
+  save: 'Save Changes',
+  saving: 'Saving...',
+  agentenabled: 'Agent Enabled',
+  botAPI: 'Bot API',
+  chatId: 'Chat Id',
+  validationBotAPIRequired: 'You must provide a Bot API key.',
+  validationChatIdRequired: 'You must provide a Chat id.',
+  telegramsettingssaved: 'Telegram notification settings saved!',
+  telegramsettingsfailed: 'Telegram notification settings failed to save.',
+  testsent: 'Test notification sent!',
+  test: 'Test',
+});
+
+const NotificationsTelegram: React.FC = () => {
+  const intl = useIntl();
+  const { addToast } = useToasts();
+  const { data, error, revalidate } = useSWR(
+    '/api/v1/settings/notifications/telegram'
+  );
+
+  const NotificationsTelegramSchema = Yup.object().shape({
+    botAPI: Yup.string().required(
+      intl.formatMessage(messages.validationBotAPIRequired)
+    ),
+    chatId: Yup.string().required(
+      intl.formatMessage(messages.validationChatIdRequired)
+    ),
+  });
+
+  if (!data && !error) {
+    return <LoadingSpinner />;
+  }
+
+  return (
+    <Formik
```

Comment:
Maybe we can add the alert component above this similar to how I setup Slack so we can write a small guide on how to get a bot api key and chat id?